src/CLI/commands/init.ts

```diff
@@ -191,8 +191,9 @@ async function init(argv: yargs.Arguments<InitOptions>, mode: InitMode) {
 		},
 	]);
 
-	await benchmark("Initializing..", async () => {
-		const selectedPackageManager = packageManagerCommands[packageManager];
+	const selectedPackageManager = packageManagerCommands[packageManager];
+
+	await benchmark("Initializing package..", async () => {
```

Comment:
```suggestion
	await benchmark("Initializing project..", async () => {
```

Reply:
This isn't the project, but the npm package. However I do agree the current wording may be confusing for game projects. Any suggestions?

Reply:
`Initializing package.json..` maybe?

Reply:
What about `Initializing npm package..`?